src/types/message_entity.rs

```diff
@@ -58,9 +58,16 @@ pub enum MessageEntityKind {
     Bold,
     Italic,
     Code,
-    Pre { language: Option<String> },
-    TextLink { url: reqwest::Url },
-    TextMention { user: User },
+    Pre {
+        #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
We usually use `#[serde_with_macros::skip_serializing_none]` annotation, instead of this. Can you add that annotation to the enum (and remove this addition)?